### PR TITLE
Validate config 160

### PR DESCRIPTION
**Pull request by vector67**

### CODE REVIEW

File: backend/tests/testInitSpec.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ block that contains tests connected with that feature. */
 */
 
 const path = require('path');
-process.env.NODE_PATH = path.join(__dirname, '../backend');
+process.env.NODE_PATH = path.join(__dirname, '../');
```
Comment (SpellBreakerZA) [inline backend/tests/testInitSpec.js]: For consistency can we remove the slash

File: backend/tests/testStaticSpec.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 const path = require('path');
-process.env.NODE_PATH = path.join(__dirname, '../backend');
+process.env.NODE_PATH = path.join(__dirname, '../');
```
Comment (SpellBreakerZA) [inline backend/tests/testStaticSpec.js]: And here

File: backend/tests/testMySqlSpec.js [hunk 3]
```diff
@@ -1,8 +1,8 @@
 const path = require('path');
-process.env.NODE_PATH = path.join(__dirname, '../backend');
+process.env.NODE_PATH = path.join(__dirname, '../');
```
Comment (SpellBreakerZA) [inline backend/tests/testMySqlSpec.js]: Same here

**Comment by vector67** [comment]
> Ok. But I still want a standalone test suite so that if this changes, there is something to test off of. In other words if the config format changes, we want to work on validation tests separately from code generation tests (dividing out the work for example).

Do I have to do that in this branch?

**Comment by SpellBreakerZA** [comment]
> > Ok. But I still want a standalone test suite so that if this changes, there is something to test off of. In other words if the config format changes, we want to work on validation tests separately from code generation tests (dividing out the work for example).
> 
> Do I have to do that in this branch?

You don't have to...

**Pull request merged**